package.json

```diff
@@ -78,5 +78,6 @@
     "overrides": {
       "npm-run-path": "5.3.0"
     }
-  }
+  },
+  "packageManager": "pnpm@9.15.4+sha512.b2dc20e2fc72b3e18848459b37359a32064663e5627a51e4c74b2c29dd8e8e0491483c3abb40789cfd578bf362fb6ba8261b05f0387d76792ed6e23ea3b1b6a0"
```

Comment:
@oliverroick why do we need this? I had to remove this line in order to run locally.

Reply:
I was on the wrong node version this is added automatically when  you run `pnpm install` in newer node versions that use corepack.

We can prevent this by pinning the required node version to 20 and add engine-strict=true. That way it will throw and error when you use a different node version. What do you think?

Reply:
@oliverroick I'm used to running `nvm i` when I switch to the project, but it makes sense to pin the version in package.json. But maybe as an enhancement in a separated PR?